### DISCUSSION TITLE
Crowdstrike Incident

### DISCUSSION THREAD

**Discussion by drwhite26** [community/community#133414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Did GITHUB release a statement on whether they were impacted by the Crowdstrike incident?

**Comment by owencmcgrath** [comment]
It doesn't look like they have. 

If they do, you might find it [here](https://github.com/about/press).

Have a nice day!

**Comment by tszdabee** [comment]
@drwhite26 No
